Pod/Classes/ProductRecommendationModel.swift

```diff
@@ -34,13 +34,14 @@ public class ProductRecommendationModel: NSObject, NSCopying {
     public var currency: String?
     public var specialPrice: NSNumber?
     public var specialPriceText: String?
+    public var region: String?
 
     public func copy(with zone: NSZone? = nil) -> Any {
         let copy = ProductRecommendationModel(productId: productId, name: name, inStock: inStock, url: url, mUrl: mUrl,
                 image: image, imageXS: imageXS, imageS: imageS, imageM: imageM, imageL: imageL, imageXL: imageXL, category: category,
                 categories: categories, brand: brand, price: price, oldPrice: oldPrice, gender: gender, colors: colors, sizes: sizes,
                 labels: labels, noUpdate: noUpdate, params: params, priceText: priceText, oldPriceText: oldPriceText,
-                language: language, currency: currency, specialPrice: specialPrice, specialPriceText: specialPriceText)
+                language: language, currency: currency, specialPrice: specialPrice, specialPriceText: specialPriceText, region:region)
```

Comment:
[Product object](https://github.com/segmentify/segmentify-v3/blob/dev/commons/src/main/java/com/segmentify/lotr/commons/model/product/Product.java) does not have region field. I think we do not need this change.